NMD.pm

```diff
@@ -120,19 +136,24 @@ sub variant_exon_check {
   my $tr = $tva->transcript;
   my $vf_end = $vf->seq_region_end;
   my @exons = @{ $tr->get_all_Exons }; 
+
   # to get the last exon 
+
   my $last_exon = $exons[-1]; 
+  
   # to get the second to the last exon (penultimate exon)
   my $second_last_exon = $exons[-2];
-  my $coding_region_end = $last_exon->coding_region_end($tr);
-  my $coding_region_start = $last_exon->coding_region_start($tr);
+  my $coding_region_end = $last_exon->end;
+  my $coding_region_start = $last_exon->start;
   if (defined($coding_region_start) && $vf_end >= $coding_region_start && $vf_end <= $coding_region_end){
     return 1; 
   }
   # to check if the second to the last exon exists 
   if (defined($second_last_exon)){
-    my $coding_region_end = $second_last_exon->coding_region_end($tr);
-    if (defined($coding_region_end) && $vf_end >= $coding_region_end - (51) ){
+    # this method has been changed because of the rrverse strand consideration
```

Comment:
```suggestion
    # this method has been changed because of the reverse strand consideration
```

Everything now works fine 👍 , here is a small typo.